contentcuration/contentcuration/frontend/shared/views/details/Details.vue

```diff
@@ -297,6 +301,10 @@
       },
     },
     computed: {
+      defaultText() {
+        // Making this a computed property so it's easier to update
+        return '---';
+      },
```

Comment:
In Kolibri we have a specific recurring pattern for "empty placeholders" or "optional text" where a wide, grey dash is inserted.

There have been some updates on this pattern s this could be a good place for follow-up standardization in the design system.

Refs:

* https://kolibri-design-system.netlify.app/kemptyplaceholder/
* https://github.com/learningequality/kolibri/pull/7472


cc @jonboiser @nucleogenesis fo later

Reply:
Made a note of it here https://github.com/learningequality/studio/issues/2070